modules/travel_pay/app/controllers/travel_pay/claims_controller.rb

```diff
@@ -5,6 +5,7 @@ class ClaimsController < ApplicationController
     def index
       veis_token = client.request_veis_token
 
+      authorize
```

Comment:
instead of doing this here, can we add a `before_action` in the `TravelPay::ApplicationController`? Then we don't have to repeatedly call `authorize` every time (we'll want it on by default anyway).

Reply:
+1 on this

Reply:
done ([9b51010](https://github.com/department-of-veterans-affairs/vets-api/pull/16231/commits/9b51010c4ea83a3d8ded511d7a03b0cdd066e4a3))